quartz/plugins/transformers/ofm.ts

```diff
@@ -532,6 +539,24 @@ export const ObsidianFlavoredMarkdown: QuartzTransformerPlugin<Partial<Options>
         })
       }
 
+      if (opts.enableCheckbox) {
+        plugins.push(() => {
+          return (tree: HtmlRoot) => {
+            visit(tree, "element", (node) => {
+              if (node.tagName === "input" && node.properties.type === "checkbox") {
+                const isChecked = node.properties?.checked ?? false
+                const id = randomUUID()
```

Comment:
this will mean that each checkbox gets a new `UUID` when a new build happens, I wonder if there's something more stable we can use here

Reply:
Yeah, I didn't think about that. I just committed another idea to handle this, just let me know if it makes any sense!. Thank you